addons/api/addon/models/session.js

```diff
@@ -102,6 +102,13 @@ export default class SessionModel extends GeneratedSessionModel {
     return this.isActive || this.isPending;
   }
 
+  /**
+   * True if status is an unknown type.
+   * @type {boolean}
+   */
+  get isUnknown() {
```

Comment:
If sessions come in different types, this is ambiguous.  Let's clarify:
```suggestion
  get isUnknownStatus() {
```

Reply:
Recommending this because we use `isUnknown` elsewhere to mean "unknown type".  But in this context we mean status.  Arguably we should be more specific about both.  For now let's change this instance.